keyboards/mechmini/v2/keymaps/wsturgiss/keymap.c

```diff
@@ -0,0 +1,98 @@
+#include QMK_KEYBOARD_H
+#include "../../../../../quantum/rgblight_list.h"
+
+#define base 0
+#define raise 1 
+#define lower 2
+
+//Tap Dance Declarations
+enum {
+  TD_SEMI_QUOT = 0,
+  TD_COMM_MINUS = 1,
+  TD_DOT_EQUAL = 2,
+  TD_SLASH_BACKSLASH = 3,
+  TBL_FLP = SAFE_RANGE,
+  RST
+};
+
+//Tap Dance Definitions
+qk_tap_dance_action_t tap_dance_actions[] = {
+  //Tap once for ;, twice for ' -not used anymore
+  [TD_SEMI_QUOT]  = ACTION_TAP_DANCE_DOUBLE(KC_SCLN, KC_QUOT),
+  //Tap once for , twice for -
+  [TD_COMM_MINUS]  = ACTION_TAP_DANCE_DOUBLE(KC_COMM, KC_MINUS),
+  //Tap once for . twice for =
+  [TD_DOT_EQUAL]  = ACTION_TAP_DANCE_DOUBLE(KC_DOT, KC_EQUAL),
+  //Tap once for / twice for '\'
+  [TD_SLASH_BACKSLASH]  = ACTION_TAP_DANCE_DOUBLE(KC_SLSH, KC_BSLS)
+};
+
+#define MODS_CTRL_MASK  (MOD_BIT(KC_LSHIFT)|MOD_BIT(KC_RSHIFT))
```

Comment:
You don't appear to be using this anywhere.

```suggestion
```

There are also macros for this in the core code; you don't need to define them in your keymap.

https://github.com/qmk/qmk_firmware/pull/4337